scripts/10_changing_logic_for_visits.sql

```diff
@@ -0,0 +1,49 @@
+USE `Soft`;
+
+ALTER TABLE `Visits`
+    ADD    `MarkID` 	       BIGINT UNSIGNED,
+    ADD CONSTRAINT    `FK_MarkID`    FOREIGN KEY (`MarkID`)    REFERENCES `Marks` (`ID`);
```

Comment:
It is better to add constraint only after MarkID values for old entries have been filled. Otherwise, there would be a problem with consistency.